src/Datadog.Trace.Core/Logging/LogEventLevel.cs

```diff
@@ -0,0 +1,12 @@
+namespace Datadog.Trace
```

Comment:
This seemed like the easiest way to allow log level checks, it will require an update to the vendors tool, to ignore that file on update.